cmake/pika_setup_hip.cmake

```diff
@@ -21,6 +22,10 @@ if(PIKA_WITH_HIP AND NOT TARGET roc::hipblas)
   # Setup hipblas (creates roc::hipblas)
   find_package(hipblas REQUIRED HINTS $ENV{HIPBLAS_ROOT} CONFIG)
 
+  # Tmp! Until hipSOLVER is complete for DLA-Future
+  find_package(rocblas REQUIRED HINTS $ENV{ROCBLAS_ROOT})
+  find_package(rocsolver REQUIRED HINTS $ENV{ROCSOLVER_ROOT})
+
```

Comment:
Together with this, it is required also to add `roc::rocblas` and `roc::rocsolver` here

https://github.com/pika-org/pika/blob/a5c06c5146b3a1618796b50272dc8fdc063a893a/libs/pika/async_cuda/CMakeLists.txt#L50-L51